plugin/PDFView.qml

```diff
@@ -190,7 +198,41 @@ SilicaFlickable {
 
     children: [
         HorizontalScrollDecorator { color: Theme.highlightDimmerColor },
-        VerticalScrollDecorator { color: Theme.highlightDimmerColor }
+        VerticalScrollDecorator { color: Theme.highlightDimmerColor },
+        PDFContextMenu {
+            id: contextMenuLinks
+            //% "External link"
+            title: qsTrId("sailfish-office-tl-pdf-link")
+            MenuItem {
+                text: contextMenuLinks.message.indexOf("mailto:") === 0 ?
+                //% "Open mail composer"
```

Comment:
Problem now is that email application might not be installed. And all the other urls are now considered Browser's responsibility, e.g. tel:12345.

How if we have special case for web urls http/https with the current text, and all the others would be "Open externally", being then handled with whatever is used. Checking current handler for mime type and mapping them to application names can be done later if really really needed.